src/quantize/tables.rs

```diff
@@ -240,3 +184,24 @@ const ac_qlookup_12_Q3_raw: [u16; QINDEX_RANGE] = [
   22766, 23214, 23662, 24126, 24590, 25070, 25551, 26047, 26559, 27071, 27599,
   28143, 28687, 29247,
 ];
+
+/// # Panics
+///
+/// - If any value is zero
+#[allow(clippy::let_unit_value)]
+#[allow(clippy::unused_unit)]
+const fn nonzero_checked(
+  raw: [u16; QINDEX_RANGE],
+) -> [NonZeroU16; QINDEX_RANGE] {
+  // SAFETY: We initialize everything before exiting this function
+  unsafe {
+    let mut nonzero = [MaybeUninit::uninit(); QINDEX_RANGE];
+    let mut i = 0;
+    while i < QINDEX_RANGE {
+      let _ = if raw[i] == 0 { [(); 0][i] } else { () };
```

Comment:
If the `ac_qlookup_12_Q3_raw` array contained a zero at index 190 for example, we would receive a compile-time error:
```
error[E0080]: could not evaluate static initializer
   --> src/quantize/tables.rs:201:32
    |
35  |   nonzero_checked(ac_qlookup_12_Q3_raw);
    |   ------------------------------------- inside `tables::ac_qlookup_12_Q3` at src/quantize/tables.rs:35:3
...
201 |       let _ = if raw[i] == 0 { [(); 0][i] } else { () };
    |                                ^^^^^^^^^^
    |                                |
    |                                index out of bounds: the length is 0 but the index is 190
    |                                inside `nonzero_checked` at src/quantize/tables.rs:201:32
```